src/main/scala/com/sksamuel/scapegoat/inspections/collections/PredefSeqIsMutable.scala

```diff
@@ -1,27 +1,29 @@
 package com.sksamuel.scapegoat.inspections.collections
 
-import com.sksamuel.scapegoat.{ Levels, Inspection, InspectionContext, Inspector }
+import com.sksamuel.scapegoat.{ Levels, Inspection, InspectionContext, Inspector, isScala213 }
 
 /** @author Stephen Samuel */
 class PredefSeqIsMutable extends Inspection("Predef.Seq is mutable", Levels.Info,
   "Predef.Seq aliases scala.collection.mutable.Seq. Did you intend to use an immutable Seq?") {
 
   def inspector(context: InspectionContext): Inspector = new Inspector(context) {
-    override def postTyperTraverser = Some apply new context.Traverser {
+    override def postTyperTraverser = if (isScala213) None else Some(
```

Comment:
`Option` should be better here, because `Some(null)` gives `null` as a value if you try to get it from the box.

Reply:
That is only useful when wrapping an expression which potentially is `null`. I don't see how the `new context.Traverser { ... }` expression can ever end up in being `null`.

Reply:
It's useful as a general practice, I think, just to save some brain cycles and let the compiler do some work for you.
Otherwise, I completely agree that there are no obvious/probable scenarios where the expression can be null.